reports/Arkansas.md

```diff
@@ -1,19 +1,28 @@
 ## Bentonville
 
-### Law enforcement gas a crowd chanting “we want peace” right after exiting the building. | June 1st
+### Law enforcement tear-gas a crowd chanting “we want peace” right after exiting the building. | June 1st
 
-Law enforcement gas a crowd chanting “we want peace” right after exiting the building, in the video there was no order to disperse, the crowd seemed to be peacefully chanting.
-Article says crowd started throwing water bottles and firecrackers. Sheriff's office claims rocks were thrown.
+A crowd gathered on the Bentonville Square to protest. Around 9 PM, law enforcement gassed a crowd chanting “we want peace.” Local news says some people throwing water bottles and firecrackers. Police also claim rocks were thrown. Police claim that at least one police vehicle was damaged. Local news reports vary -- some reports have protesters "kicking" a police vehicle; some televised reports have protesters "damaging" a police vehicle; most written articles merely report that police say protesters damaged a police vehicle. Assuming that a police vehicle was in fact damaged, it is unclear to what extent (though per one op-ed writer, the police chief says $8000 worth of damage was done).
 
-tags: tear-gas, tear-gas-canister
+It is unclear whether any order to disperse was given before the first use of tear gas; in the videos purporting to be from that time, no such order is audible. (In one video -- see third link -- a voice can be heard that sounds like it comes from a PA system, but the words are unclear.) Several people present, including one op-ed writer, claim not to have heard any such order. Orders to disperse were certainly given repeatedly between 9:00 and 11:00, as police continued to deploy tear gas.
```

Comment:
> Orders to disperse were certainly given repeatedly between 9:00 and 11:00, as police continued to deploy tear gas.

Wait, how do you know this?

Reply:
There's at least one video of an order to disperse very clearly being given (though it's unclear at what time it was taken) -- see the last video in the 5 News report. 5 News says twice that protesters were repeatedly told to disperse. 40/29 News doesn't say how many dispersal orders were given, but specifically references two separate such orders. I didn't see any source denying that these later orders were given. The use of tear gas, after the first volley, is also a point of agreement among all the reports I saw.